src/services/app-config.service.js

```diff
@@ -81,3 +84,107 @@ export function deleteApp() {
     }
   );
 }
+
+export async function fetchAndStoreAppConfig() {
+  const apps = await fetchAllApps();
+  if (apps.data.length) {
+    store.dispatch("updateAppConfigurationStatus", true);
+    const currentApp = apps.data[0];
+    const appId = currentApp.ID;
+    store.dispatch("updateAppName", currentApp.name);
+    store.dispatch("updateAppId", currentApp.ID);
+
+    // Get Address
+    const appAddress = await getAddress(currentApp.address);
+    store.dispatch("updateSmartContractAddress", appAddress);
+
+    const env = store.getters.env;
+    const chainType = ["ethereum", "polygon", "binance"][currentApp.chain];
+    store.dispatch(env + "/updateChainType", chainType);
+    const unlimitedBytes = 10995116277760;
+
+    if (currentApp.storage_limit < unlimitedBytes) {
+      let storage;
+
+      // Convert to MB if storage is less than 1023 MB
+      // else convert to GB
+      if (currentApp.storage_limit <= bytes("1023MB")) {
+        storage = bytes(currentApp.storage_limit, {
+          unitSeparator: " ",
+          unit: "MB",
+        });
+      } else {
+        storage = bytes(currentApp.storage_limit, {
+          unitSeparator: " ",
+          unit: "GB",
+        });
```

Comment:
Because both branches of this `if/else` only differ in the unit, it might be better to move the condition inline.

```js
const isUnder1GB = currentApp.storage_limit < bytes("1GB");
storage = bytes(currentApp.storage_limit, {
  unitSeparator: " ",
  unit: isUnder1GB ? "MB" : "GB",
});
```